src/qt/intro.cpp

```diff
@@ -345,3 +329,24 @@ QString Intro::getPathToCheck()
     mutex.unlock();
     return retval;
 }
+
+void Intro::UpdatePruneLabels(int64_t prune_target_gb)
+{
+    QString storageRequiresMsg = tr("At least %1 GB of data will be stored in this directory, and it will grow over time.");
+    if (prune_target_gb) {
+        if (prune_target_gb <= m_blockchain_size_gb) {
```

Comment:
In commit "refactor: Add Intro::UpdatePruneLabels()" (9759ccbb40a1ce333d8ae3b6c3ed82520a948b76)

It seems like this should set `m_required_space_gb = m_blockchain_size_gb + m_chain_state_size_gb` in the case where `prune_target_gb > m_blockchain_size_gb` instead of not updating `m_required_space_gb`